changeset:   433:6a7c398b04a7
user:        Petr Vandrovec <petr@vandrovec.name>
date:        Sun Jul 03 20:36:50 2005 +0100
files:       .hgignore
description:
Switch from BitKeeper to Mercurial.  Get rid of BitKeeper config
files, and introduce Mercurial's .hgignore.


diff -r 93a411983bcc -r 6a7c398b04a7 .hgignore
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/.hgignore	Sun Jul 03 20:36:50 2005 +0100
@@ -0,0 +1,179 @@
+\.a$
+\.d$
+\.do$
+\.la$
+\.lo$
+\.o$
+\.so$
+^Make\.rules$
+^Makefile$
+^autom4te.cache/
+^config\.log$
+^config\.status$
+^contrib/pam/Makefile$
+^contrib/php/Makefile$
+^contrib/php/Makefile\.in$
+^contrib/php/\.deps$
+^contrib/php/\.libs/
+^contrib/php/config\.h$
+^contrib/php/config\.log$
+^contrib/php/config\.nice$
+^contrib/php/config\.status$
+^contrib/php/config_vars\.mk$
+^contrib/php/libs\.mk$
+^contrib/php/php_auth_nds\.slo$
+^contrib/tcl-utils/Makefile$
+^contrib/testing/Makefile$
+^contrib/testing/abbrev$
+^contrib/testing/canon$
+^contrib/testing/chgpwd$
+^contrib/testing/cmpattr$
+^contrib/testing/copyauth$
+^contrib/testing/dirlimit$
+^contrib/testing/dirlist2$
+^contrib/testing/dssearch$
+^contrib/testing/dsstream$
+^contrib/testing/eaops$
+^contrib/testing/effright$
+^contrib/testing/fileinfo$
+^contrib/testing/large$
+^contrib/testing/lic$
+^contrib/testing/mkattr$
+^contrib/testing/mkclass$
+^contrib/testing/mkfile$
+^contrib/testing/mkobj$
+^contrib/testing/modclass$
+^contrib/testing/modobj$
+^contrib/testing/mvobj$
+^contrib/testing/nfssetinfo$
+^contrib/testing/nsfileinfo$
+^contrib/testing/nwbcast$
+^contrib/testing/nwdsgethost$
+^contrib/testing/nwdsgetstat$
+^contrib/testing/nwdsgetvi$
+^contrib/testing/nwdslist$
+^contrib/testing/nwdspartjoin$
+^contrib/testing/nwdspartlist$
+^contrib/testing/nwdspartrm$
+^contrib/testing/nwdspartsplit$
+^contrib/testing/nwdsresetstat$
+^contrib/testing/nwgettime$
+^contrib/testing/partroot$
+^contrib/testing/physlock$
+^contrib/testing/pp/Makefile$
+^contrib/testing/pp/bcastmd$
+^contrib/testing/pp/getconn_addr$
+^contrib/testing/pp/getsynt$
+^contrib/testing/pp/nwc_next_object$
+^contrib/testing/pp/nwgetconnlist$
+^contrib/testing/pp/nwwhoami$
+^contrib/testing/pp/open_conn_by_name$
+^contrib/testing/pp/readsdef$
+^contrib/testing/pp/readsynt$
+^contrib/testing/pp/return_block_of_tree$
+^contrib/testing/pp/scantree$
+^contrib/testing/pp/scanvolr$
+^contrib/testing/pp/semaphor$
+^contrib/testing/pp/treelist$
+^contrib/testing/pp/vlist$
+^contrib/testing/pp/volres$
+^contrib/testing/readaddr$
+^contrib/testing/readadef$
+^contrib/testing/readattr$
+^contrib/testing/readcc$
+^contrib/testing/readcls$
+^contrib/testing/readsynt$
+^contrib/testing/reloadds$
+^contrib/testing/renobj$
+^contrib/testing/replops$
+^contrib/testing/resint$
+^contrib/testing/resolve$
+^contrib/testing/rmattr$
+^contrib/testing/rmclass$
+^contrib/testing/rmobj$
+^contrib/testing/schsync$
+^contrib/testing/semaphore$
+^contrib/testing/setkeys$
+^contrib/testing/time$
+^contrib/testing/timegv$
+^contrib/testing/timeinfo$
+^contrib/testing/verpwd$
+^contrib/testing/verunix$
+^contrib/testing/vollist$
+^contrib/testing/wcsmbs-tst1$
+^contrib/tests/Makefile$
+^contrib/tknwmsg/Makefile$
+^include/config\.h$
+^include/ncp/ext/socket\.h$
+^include/ncp/ext/stdint\.h$
+^include/ncp/kernel/fs\.h$
+^include/ncp/kernel/if\.h$
+^include/ncp/kernel/ipx\.h$
+^include/ncp/kernel/route\.h$
+^include/ncp/kernel/types\.h$
+^include/ncp/nwnet\.h$
+^intl/Makefile$
+^ipx-1\.0/.*\.gz$
+^ipx-1\.0/Makefile$
+^ipx-1\.0/Samples/Makefile$
+^ipx-1\.0/Samples/ipxrcv$
+^ipx-1\.0/Samples/ipxsend$
+^ipx-1\.0/Samples/rip$
+^ipx-1\.0/Samples/sap$
+^ipx-1\.0/ipx_cmd$
+^ipx-1\.0/ipx_configure$
+^ipx-1\.0/ipx_interface$
+^ipx-1\.0/ipx_internal_net$
+^ipx-1\.0/ipx_route$
+^ipxdump/Makefile$
+^lib/Makefile$
+^lib/libncp\.so\..*$
+^man/.*\.gz$
+^man/Makefile$
+^po/Makefile$
+^po/Makefile\.in$
+^po/POTFILES$
+^sutil/Makefile$
+^sutil/ncplogin$
+^sutil/ncpmap$
+^sutil/ncpmount$
+^sutil/ncpumount$
+^sutil/nwsfind$
+^util/Makefile$
+^util/ncopy$
+^util/ncptest$
+^util/nprint$
+^util/nsend$
+^util/nwauth$
+^util/nwbocreate$
+^util/nwbols$
+^util/nwboprops$
+^util/nwborm$
+^util/nwbpadd$
+^util/nwbpcreate$
+^util/nwbprm$
+^util/nwbpset$
+^util/nwbpvalues$
+^util/nwdir$
+^util/nwdpvalues$
+^util/nwfsctrl$
+^util/nwfsinfo$
+^util/nwfstime$
+^util/nwgrant$
+^util/nwlistsalvage$
+^util/nwmsg$
+^util/nwpasswd$
+^util/nwpjmv$
+^util/nwpqjob$
+^util/nwpurge$
+^util/nwrevoke$
+^util/nwrights$
+^util/nwsalvage$
+^util/nwtrustee$
+^util/nwtrustee2$
+^util/nwuserlist$
+^util/nwvolinfo$
+^util/pqlist$
+^util/pqstat$
+^util/pserver$
+^util/slist$
